### ISSUE TITLE
Jittery controls on Audio Unit in Live 12

### ISSUE THREAD

**Issue by Sasha-Radojevic** [JeffMcClintock/SynthEdit-Tracking#69]
I'm a bit confused with [your response here](https://groups.io/g/synthedit/message/27997) because VST3 works fine, but it is AU that still has jittery controls bug.

Here are the test plugins:
https://www.dropbox.com/scl/fo/hqxojvfgpkwhzpa9hid11/AOlwEEQynoc-dQGaHPzii-s?rlkey=omz3ys41p685x8ym4ljubjv01&dl=0

**Comment by JeffMcClintock** [comment]
I tested SynthEdit in Ableton Live 11 on macOS and did experience jitter in the controls.
My experience was that when I moved a knob rapidly, it briefly "jumps back" to its starting position, then resumes its movement correctly.

I printed a log of this behavior. What happens is when you first 'grab' (mouse-down) the knob, Ableton queries the knob value (2nd line of log).   Then some time later, Ableton calls setParameter on the plugin with that initial value (line 7 of log) This is the 'jump back' (jitter) that I observed.

This appears to be a bug in Ableton, which we know has similar bugs with VST3 plugins. I can't fix bugs in Ableton.

-----LOG---------
ParamGrabbed(1) 38
getValueImmediate 0.000000
DAW <= AUParameterSet(0.00449219)
DAW => SetParameter(38, 0.00449219)
DAW <= AUParameterSet(1.04746)
DAW => SetParameter(38, 1.04746)
DAW => SetParameter(38, 0)
getValueImmediate 0.000000
DAW <= AUParameterSet(7.27676)
DAW => SetParameter(38, 7.27676)
DAW => ParameterListener(38, 0)
setRealFromDaw 0.000000
DAW <= AUParameterSet(10)
DAW => SetParameter(38, 10)
ParamGrabbed(0) 38
